e2e_test/streaming/now.slt

```diff
@@ -0,0 +1,27 @@
+system ok
+psql -h $RISEDEV_RW_FRONTEND_LISTEN_ADDRESS -p $RISEDEV_RW_FRONTEND_PORT -U root -d dev -c "
+create materialized view mv as
+select * from generate_series(
+    to_timestamp($(date +%s) - 10),
```

Comment:
This is clever but it would be better if we can support specifying `NOW() - 10 second` here. 😄

Reply:
They are not exactly same semantic. 
generate_series(now() - 10 sec, now(), step)` is equal to the statment with a temporal filter
```
create materialized view mv as
select * from (generate_series(${now_when_create_statment} - 10 sec, now(), step))
where generate_series > now() - 10 sec
```
So we do not have a `current_now()` or `now_at_the_statment()` function 😆

Reply:
> So we do not have a `current_now()` or `now_at_the_statment()` function 😆

PostgreSQL has 4 different "now"s: #2870

* For `start`, if we really want time of `create`, it shall be `timestamptz 'now'`.
* For `stop`, if we really want endless stream, it shall be `timestamptz 'infinity'`.

Reply:
> * For `stop`, if we really want endless stream, it shall be `timestamptz 'infinity'`.

But seems `infinity` cannot clearly convey the idea that the timestamp is emitted along with the current epoch🤔

Reply:
It seems that we should've introduced something like `streaming_now()` for the use of dynamically updated `now` in streaming mode. So that `now()` won't be so confusing when it appears with 2 semantics in one statement.

Reply:
When we use `now()` to express the semantics of a temporal filter, `now()` represents a constantly changing timestamp within a streaming query. Therefore, it becomes difficult to express a semantics of the current value of now() that does not change later.